SDWebImage/Core/SDDiskCache.m

```diff
@@ -165,7 +165,7 @@ - (void)removeExpiredData {
     NSArray<NSString *> *resourceKeys = @[NSURLIsDirectoryKey, cacheContentDateKey, NSURLTotalFileAllocatedSizeKey];
     
     // This enumerator prefetches useful properties for our cache files.
-    NSDirectoryEnumerator *fileEnumerator = [self.fileManager enumeratorAtURL:diskCacheURL
+    NSDirectoryEnumerator<NSURL *> *fileEnumerator = [self.fileManager enumeratorAtURL:diskCacheURL
                                                includingPropertiesForKeys:resourceKeys
                                                                   options:NSDirectoryEnumerationSkipsHiddenFiles
```

Comment:
Why this does not use `NSDirectoryEnumerationProducesRelativePathURLs` ?

Reply:
according to the [comment below](https://github.com/SDWebImage/SDWebImage/pull/3690#issuecomment-2018205511), will delete the option .